packages/react-debug-tools/src/ReactDebugHooks.js

```diff
@@ -356,6 +356,23 @@ const Dispatcher: DispatcherType = {
   useId,
 };
 
+// create a proxy to throw a custom error
+// in case future versions of React adds more hooks
+const DispatcherProxyHandler = {
+  get(target, prop) {
+    if (target.hasOwnProperty(prop)) {
+      return target[prop];
+    }
+    const error = new Error('Missing method in Dispatcher: ' + prop);
+    // Note: This error name needs to stay in sync with react-devtools-shared
+    // TODO: refactor this if we ever combine the devtools and debug tools packages
+    error.name = 'UnsupportedFeatureError';
```

Comment:
Sorry, I missed this! Could you make this error something more package specific (ex. 'ReactDebugToolsUnsupportedFeatureError') to minimize name clashes?

Reply:
How about `ReactDebugToolsError` (for more consistency with built in error names)